src/renderer/components/+namespaces/namespace-select.tsx

```diff
@@ -28,17 +29,13 @@ const defaultProps: Partial<Props> = {
 @observer
 export class NamespaceSelect extends React.Component<Props> {
   static defaultProps = defaultProps as object;
-  private unsubscribe = noop;
 
-  async componentDidMount() {
-    if (!namespaceStore.isLoaded) {
-      await namespaceStore.loadAll();
-    }
-    this.unsubscribe = namespaceStore.subscribe();
-  }
-
-  componentWillUnmount() {
-    this.unsubscribe();
+  componentDidMount() {
+    disposeOnUnmount(this, [
+      kubeWatchApi.subscribeStores([namespaceStore], {
+        preload: !namespaceStore.isLoaded,
+      })
```

Comment:
Given that `subscribeStores` only has complete oversite it shouldn't erroneously load this value again if it was recently loaded. We shouldn't have to do this sort of fine control outside of `kubeWatchApi`

Reply:
Again, same logic with loading was there before PR and I don't see any reason to change it.

Reply:
updated to `preload: true`